cgnsutilities/cgnsutilities.py

```diff
@@ -3168,7 +3172,7 @@ def combineGrids(grids, useOldNames=False):
     # Create a dictionary to contain grid objects with their names
     # as the corresponding keys
     gridDict = {}
-    for j, grid in enumerate(grids):
+    for _j, grid in enumerate(grids):
```

Comment:
If the variable `j` was unused, then we can simply loop with `for grid in grids:` instead right?

Reply:
Good point